docs/content/concepts/tokenomics/sui-bridging.mdx

```diff
@@ -112,6 +112,81 @@ See the [Wormhole FAQ](https://wormhole.com/docs/products/connect/overview/) for
+
+### Supported assets
+
+ZetaChain supports native cross-chain transfers for the following tokens:
+
+- **Bitcoin (BTC)** - Native Bitcoin transfers via HTLC
+- **Ethereum (ETH)** - Native ETH and ERC-20 tokens including USDC, USDT, WBTC
+- **Polygon (MATIC)** - Native MATIC and Polygon tokens
+- **BSC (BNB)** - Native BNB and BEP-20 tokens
+- **Avalanche (AVAX)** - Native AVAX and Avalanche C-Chain tokens
+- **Layer 2 chains** - Arbitrum, Optimism, Base
+
+Assets bridged through ZetaChain are represented as ZRC-20 tokens on the ZetaChain network and can be transferred to any connected chain.
+
+See the [ZetaChain supported networks](https://www.zetachain.com/docs/developers/chains/list) for the complete list.
+
+### Omnichain messaging
+
+Beyond asset transfers, ZetaChain's Universal App framework enables smart contracts to communicate across all connected chains. Developers can build applications that:
+
+- Call functions on contracts across multiple blockchains
+- Read and write state across chains through ZetaChain's validators
+- Execute complex multi-chain workflows in a single transaction
+
+### How it works
+
+ZetaChain uses a proof-of-stake validator network to observe transactions on external chains and relay messages between them. When you initiate a cross-chain transaction:
+
+1. You interact with a Gateway contract on the source chain (Sui or any supported chain)
+2. ZetaChain validators observe and verify the transaction
+3. The validators execute any omnichain contract logic on ZetaChain
+4. The validators trigger the corresponding action on the destination chain
+
+All cross-chain operations are handled automatically without manual intervention.
+
+### Gas handling
+
+ZetaChain provides flexible gas payment options:
+
+- **Pay on source chain** - Pay all fees upfront on the source chain in ZETA, USDC, or USDT
+- **Automatic gas drop** - Receive native tokens on the destination chain for transaction fees
+- **Custom gas logic** - Implement application-specific gas payment models
+
+### Gateway contracts
+
+Interact with ZetaChain through Gateway contracts deployed on each supported chain:
+
+**Athens Testnet**
+- Sepolia (Ethereum): `0x6c533f7fe93fae114d0954697069df33c9b74fd7`
+- Base Sepolia: `0x0c487a766110c85d301d96e33579c5b317fa4995`
+- Amoy (Polygon): `0x3B4e6cD5F8b00a23a76e1F4c36b5D3Ec50dB4D9e`
+- BSC Testnet: `0x8943738f2f21a7bE27C0749cEf35c6c1B3E4b7E1`
+
+Always verify gateway addresses from the [official ZetaChain documentation](https://www.zetachain.com/docs/reference/network/details) before integrating.
+
+### Getting started
+
+Install the ZetaChain CLI:
+
+```bash
+npm install -g @zetachain/cli
+```
+
+Explore example contracts and tutorials:
+- [Universal Apps Tutorial](https://www.zetachain.com/docs/developers/tutorials/intro)
+- [Cross-Chain Messaging Guide](https://www.zetachain.com/docs/developers/tutorials/messaging)
```

Comment:
@ExpertVagabond this style of messaging only works with EVM chains.

Instead, please, add Sui-specific tutorials:

- https://www.zetachain.com/docs/developers/tutorials/sui
- https://www.zetachain.com/docs/developers/tutorials/sui-withdraw-and-call